akka24-conductr-client-lib/src/main/scala/com/typesafe/conductr/clientlib/akka/JsonMarshalling.scala

```diff
@@ -84,9 +84,24 @@ object JsonMarshalling {
     Format(reads, writes)
   }
 
+  implicit object BundleConfigEndpointFormat extends Format[BundleConfigEndpoint] {
+    override def writes(o: BundleConfigEndpoint): JsValue =
+      throw new UnsupportedOperationException()
+
+    override def reads(json: JsValue): JsResult[BundleConfigEndpoint] =
+      for {
+        bindProtocol <- (json \ "bindProtocol").validate[String]
+        serviceName <- (json \ "serviceName").validateOpt[String]
+        services <- (json \ "services").validateOpt[Set[URI]]
+      } yield BundleConfigEndpoint(
+        bindProtocol,
+        serviceName,
+        services.getOrElse(Set.empty)
+      )
+  }
+
```

Comment:
Ah yes, thanks for picking this up.

Issue created:
https://github.com/typesafehub/conductr-lib/issues/114

Also trello card:
https://trello.com/c/5DEq06o2/369-conductr-client-lib-get-bundles-should-support-unmarshalling-of-endpoints-declared-using-acls